helix-view/src/editor.rs

```diff
@@ -913,6 +924,18 @@ impl Default for Config {
             popup_border: PopupBorderConfig::None,
             indent_heuristic: IndentationHeuristic::default(),
             jump_label_alphabet: ('a'..='z').collect(),
+            persist_old_files: false,
+            persist_commands: false,
+            persist_search: false,
+            persist_clipboard: false,
+            // TODO: any more defaults we should add here?
+            persistence_file_exclusions: [r".*/\.git/.*"]
```

Comment:
Please add `r".*/COMMIT_EDITMSG"` here. When using bare git repos, the git dir is not always named `.git` (see https://news.ycombinator.com/item?id=11070797).